packages/cli/create-strapi-starter/utils/fetch-npm-starter.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+const path = require('path');
+const execa = require('execa');
+const chalk = require('chalk');
+const stopProcess = require('./stop-process');
+
+/**
+ * Gets the package version on npm. Will fail if the package does not exist
+ * @param {string} packageName - Name to look up on npm, may include a specific version
+ * @returns {Object}
+ */
+async function getPackageInfo(packageName) {
+  const { stdout } = await execa.shell(`npm view ${packageName} name version --silent`);
```

Comment:
I think it wouldn't be necessary here, because this is not a user-facing operation: it only happens inside the script, nothing is printed on screen. And we know that npm should always be installed, so I think it's easier to use it directly

Reply:
Yarn is faster in most cases so if yarn is available it would be preferable to favor it